docs/howto/override-project-templates.txt

```diff
@@ -0,0 +1,248 @@
+==============================================================
+How to override project and app file structure using templates
+==============================================================
+
+
+Introduction
+============
+
+The first command you will run when setting up a django project is
```

Comment:
We often don't have a titled introduction and just start 
Also Django should have a capital letter :+1: 
```suggestion

The first command you will run when setting up a Django project is
```